autopts/pybtp/btp/gap.py

```diff
@@ -1420,3 +1455,35 @@ def gap_padv_sync_transfer_recv(skip, sync_timeout, flags, addr_type=None, addr=
     iutctl.btp_socket.send(*GAP['padv_sync_transfer_recv'], data=data_ba)
 
     gap_command_rsp_succ()
+
+
+def gap_conn_br(bd_addr=None):
+    logging.debug("%s %r", gap_conn_br.__name__, bd_addr)
+    iutctl = get_iut()
+
+    data_ba = bytearray()
+    bd_addr_ba = addr2btp_ba(pts_addr_get(bd_addr))
+
+    data_ba.extend(bd_addr_ba)
+
+    iutctl.btp_socket.send_wait_rsp(*GAP['conn_br'], data=data_ba)
+
+
+def gap_pair_with_sec_level(bd_addr=None, bd_addr_type=None, sec_level=defs.GAP_PAIR_LEVEL_2):
+    logging.debug("%s %r %r %r", gap_pair_with_sec_level.__name__, bd_addr, bd_addr_type, sec_level)
```

Comment:
We shouldn't have both `gap_pair_with_sec_level` and `gap_pair` IMO. 

Suggest to add `, sec_level=defs.GAP_PAIR_LEVEL_2)` to `gap_pair` instead

Reply:
I found auto-pts has been shared by `nimble`, `bluez`, `nrf-v160` and `zephyr`. I am not sure whether all of them can handle the case that a new argument is added.

So, I added a new command for security level setting.